__tests__/e2e/scripts/ci_integration.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [ -z "${JENKINS_URL}" ]; then
+  # formatting
+  bold=$(tput bold)
+  normal=$(tput sgr0)
+  SLEEP_TIME="0.1"
+else
+  SLEEP_TIME="10"
+fi
+
+# run e2e tests journey
+##################################################
+SYNTHETICS_JUNIT_FILE="junit_$1.xml" npx @elastic/synthetics synthetics.journey.ts --reporter junit
+
+# Take the stack down
+elastic-package stack down
+
+docker system prune --force
```

Comment:
I dont think this is correct, this is a disruptive command and should be used with caution. We should only prune stuff that was started by running the test.